### PR TITLE
[plugin.video.catchuptvandmore] 0.2.29

### PR DESCRIPTION
**Pull request by sy6sy2**
### Description

[Version 0.2.29]

[FR - MyTF1/LCI] Fix Live TV / Catch-Up TV
[FR - France-O] Remove Live TV
[FR - Fun Radio and RTL2] Fix Live TV
[FR - Gulli] Move Gulli to 6play
[FR - 6Play] Add Cage Warrios - Catch-Up TV
[WO - Arte] Readd Hack for Live TV
[BE - RTBF] Use HLS contents if available because DRM not supported for some kodi instances
[FR - BSmart] Fix Live TV / Catch-Up TV
[BE - TéléMB] Fix Live TV
[BE - RTC Liège] Fix Live TV

### Checklist:

- [X] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [X] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [X] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practice but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-plugins/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.
- If you see no activity on your PR after a week (so at least one weekend has passed) then please go to the #kodi-dev freenode IRC channel to reach out to the team

### CODE REVIEW

**Comment by enen92** [comment]
```ERROR: Required dependency script.module.pytz is not available in current repository```

**Comment by sy6sy2** [comment]
Strange, we use `script.module.pytz` since a long time now and the last PR (https://github.com/xbmc/repo-plugins/pull/3327) that use Pytz passed the CI.
Was Pytz suddenly dropped from Krypton add-ons repo?

Edit: Pytz is in the Krypton repo: https://mirrors.kodi.tv/addons/krypton/script.module.pytz/
What's wrong with the CI?

**Comment by enen92** [comment]
this has to wait a bit @sy6sy2 looks like we have problems on the mirrors (see #servers)

File: plugin.video.catchuptvandmore/resources/lib/channels/fr/6play.py [hunk 1]
```diff
@@ -519,159 +519,132 @@ def get_playlist_urls(plugin,
...
-        account_timestamp = json_parser["signatureTimestamp"]
-        account_signature = json_parser["UIDSignature"]
-
-        is_helper = inputstreamhelper.Helper('mpd', drm='widevine')
-        if not is_helper.check_inputstream():
-            return False
+    if plugin.setting.get_string('6play.login') == '' or\
+            plugin.setting.get_string('6play.password') == '':
+        xbmcgui.Dialog().ok(
+            'Info',
```
Comment (enen92) [inline plugin.video.catchuptvandmore/resources/lib/channels/fr/6play.py]: Hardcoded strings presented in the UI most be localized

**Review by enen92 — CHANGES_REQUESTED** [review]
Minor

**Comment by sy6sy2** [comment]
Done 😉 
Thank you enen92 for the review!

**Review by enen92 — APPROVED** [review]

**Pull request merged**